chrome/views/gmailView.js

```diff
@@ -228,8 +228,14 @@ GmailToTrello.GmailView.prototype.parseData = function() {
     var from_raw = emailName + ' <' + emailAddress + '> ' + data.time;
     var from_md = '[' + emailName + '](' + emailAddress + ') ' + data.time;  // FYI (Ace, 10-Jan-2017): [name](url "comment") is markdown syntax
 
+    // all message times in this thread
+    data.messageTimes = [];
+    $(this.selectors.messageTimes, $viewport).each(function () {
+        data.messageTimes.push(Date.parse($(this).attr('title').replace(' at ', ' ')).getTime());
+    });
+
     // subject
-    data.subject = ($(this.selectors.emailSubject, this.$root).text() || '').trim();
+    data.subject = ($(this.selectors.emailSubject, $viewport).text() || '').trim();
```

Comment:
This is part of the issue mentioned above where when I navigated from a thread to a landing page, it was selecting the thread subject line.